woocommerce/woocommerce-paypal-payments
# Change the way of checking if WooCommerce is active

**Issue**: #461

---

### Description

The detection of is WooCommerce active using `is_plugin_active()` [does not take into account mu-plugins](https://developer.wordpress.org/reference/functions/is_plugin_active//#description).
The PR will fix that by changing the way of checking if WooCommerce is active as described in [official Woo Docs](https://woocommerce.com/document/query-whether-woocommerce-is-activated/)

### Steps to Test




1. Install WooCommerce as mu-plugin
2. Install and activate WooCommerce PayPal Payments plugin
3. See error about WooCommerce is not installed
